src/Orleans/Streams/Internal/IStreamControl.cs

```diff
@@ -32,9 +32,9 @@ namespace Orleans.Streams
     internal interface IStreamControl
     {
         /// <summary>
-        /// Perform cleanup functiuons for this stream.
+        /// Perform cleanup functions for this stream.
         /// </summary>
         /// <returns>Completion promise for the cleanup operstions for this stream.</returns>
-        Task Cleanup();
+        Task Cleanup(bool cleanupProducers = true, bool cleanupConsumers = false);
```

Comment:
Agree. I used defaults because of the legacy code in VSO tests. Now I will need to port that code as well. Not a problem, will do.